Add new proc for Content View Version Comparison (#1950)
theforeman/foreman-documentation: guides/common/modules/proc_content-view-version-comparison.adoc

> [id="Content_View_Version_Comparison_{context}"]
> = Content View Version Comparison

Reviewer: ```suggestion
[id="Comparing_Content_View_Versions_{context}"]
= Comparing Content View Versions
```
Editing title to gerund form by the IBM style guide. Please also update file name

Use this procedure to compare Content View Version functionality for {Project}.

.Procedure

> * In the {Project} web UI, navigate to *Content* > *Content Views* >  then view list of versions to select two versions and click Compare.
> 

Reviewer: ```suggestion
* In the {ProjectWebUI} , navigate to *Content* > *Content Views*, then select two Content View versions and click Compare.
```

> The compare screen will have pre-selected versions in the version dropdown menus and tabs for all content types found in either version.

Reviewer: ```suggestion
The compare screen has the pre-selected versions in the version dropdown menus and tabs for all content types found in either version.
```

You can filter the results to show only the same, different, or all content types.

> You can select different versions to compare on this screen by selecting them from the version dropdowns.

Reviewer: ```suggestion
You can compare different Content View versions by selecting them from the dropdown menus.
```
